### PR TITLE
Dev astprinter

### PR DESCRIPTION
**Pull request by javierfernandes**
Take a look and see if it is enough for Mumuki

### CODE REVIEW

**Comment by javierfernandes** [comment]
See comments in #418

File: org.uqbar.project.wollok.launch/src/org/uqbar/project/wollok/launch/WollokASTAnalyzer.xtend [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+	
+	def features(EObject it) { eContents.groupBy[eContainingFeature].entrySet.map[key?.name -> elements(value, key)] }
+	
+	def elements(List<EObject> it, EStructuralFeature feature) '''«IF feature.many»[«ENDIF» «map[e|process(e)].join(",")» «IF feature.many»]«ENDIF»'''
+	
+	// *************************************
+	// ** names transformations
+	// *************************************
+	
+	val transformations = #[ [removePrefix("W")], [removeSuffix("Expression")], [removeSuffix("Declaration")], [removeSuffix("Literal")] ]
```
Comment (flbulgarelli) [inline org.uqbar.project.wollok.launch/src/org/uqbar/project/wollok/launch/WollokASTAnalyzer.xtend]: I would keep AST elements as-is. This makes the mapping between the grammar and the generates AST obvious.

Comment (flbulgarelli) [inline org.uqbar.project.wollok.launch/src/org/uqbar/project/wollok/launch/WollokASTAnalyzer.xtend]: OK, I dint't see you comments in the issue. Now I understand that you are going to rename those elements in the near future.

**Comment by flbulgarelli** [comment]
@faloi Look, this is awesome :) It is based on http://esprima.org/, so it should be easy to process like we did in the hspec/mocha runners. 

@AlvarezAriel  could we have something like this implements for the new Gobstones?

**Pull request merged**